Tribler/vlc.py

```diff
@@ -48,6 +48,8 @@
 # Used by EventManager in override.py
 from inspect import getargspec
 
+from twisted.python.win32 import WindowsError
```

Comment:
I'm pretty sure this is a builtin exception, no need to import it

https://docs.python.org/2/library/exceptions.html#exceptions.WindowsError